docs/releases/1.10.txt

```diff
@@ -165,7 +165,10 @@ Management Commands
 Migrations
 ^^^^^^^^^^
 
-* ...
+* Serialize ``enum.Enum`` instances (native 3.4+ and from enum34_ 2.7-3.3) so
```

Comment:
I think we can remove the parenthetical reference (same in topics guide). Django 1.10 only supports Python 3.4+ so this is only relevant for Python 2.7 and I think it's reasonable that readers will assume the backported version works the same.